docker/jdk17/Dockerfile

```diff
@@ -46,8 +46,8 @@ ARG VCS_REF
 ARG VERSION
 LABEL org.label-schema.build-date=$BUILD_DATE \
       org.label-schema.name="Teku" \
-      org.label-schema.description="Ethereum 2.0 Beacon Chain Client" \
-      org.label-schema.url="https://consensys.net/knowledge-base/ethereum-2/teku/" \
+      org.label-schema.description="Ethereum Consensus Client" \
+      org.label-schema.url="https://docs.teku.consensys.io/" \
```

Comment:
I think this is better link marketing wise: https://consensys.io/teku

Reply:
it had all the wrong terminology but the old link was just plain wrong... at least the doc link is maintained... its a good start and we can revisit.